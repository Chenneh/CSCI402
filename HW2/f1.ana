
==== Packets ====
p1: inter-arrival time should be 2554.317ms
    time in Q1 should be 0.004ms
    time in Q2 should be 0.022ms
    should request 518ms of service time
    time in S1 should be 534.863ms
    time in system should be 534.907ms

p2: inter-arrival time should be 4271.155ms
    time in Q1 should be 0.004ms
    time in Q2 should be 0.027ms
    should request 387ms of service time
    time in S1 should be 404.314ms
    time in system should be 404.360ms

p3: inter-arrival time should be 2150.296ms
    time in Q1 should be 0.004ms
    time in Q2 should be 0.020ms
    should request 3686ms of service time
    time in S1 should be 3702.555ms
    time in system should be 3702.594ms

p4: inter-arrival time should be 105.096ms

p5: inter-arrival time should be 484.231ms
    time in Q1 should be 0.003ms
    time in Q2 should be 0.012ms
    should request 2664ms of service time
    time in S2 should be 2680.696ms
    time in system should be 2680.731ms

p6: inter-arrival time should be 2965.725ms
    time in Q1 should be 0.003ms
    time in Q2 should be 0.014ms
    should request 10000ms of service time
    time in S2 should be 10017.362ms
    time in system should be 10017.394ms

p7: inter-arrival time should be 2645.656ms

p8: inter-arrival time should be 2133.185ms

p9: inter-arrival time should be 1859.564ms
    time in Q1 should be 0.003ms
    time in Q2 should be 0.015ms
    should request 1414ms of service time
    time in S1 should be 1424.365ms
    time in system should be 1424.399ms

p10: inter-arrival time should be 518.122ms
    time in Q1 should be 0.002ms
    time in Q2 should be 906.275ms
    should request 2562ms of service time
    time in S1 should be 2581.179ms
    time in system should be 3487.471ms

==== Statistics ====
average packet inter-arrival time = 1.9687347
average packet service time = 3.04933342857143
average number of packets in Q1 = 9.92451928433342e-07
average number of packets in Q2 = 0.039110588745785
average number of packets in S1 = 0.373130684430233
average number of packets in S2 = 0.547922267367758
average time a packet spent in system = 3.17883657142857
standard deviation for time spent in system = 3.050753746085
token drop probability = 0
packet drop probability = 0.3
