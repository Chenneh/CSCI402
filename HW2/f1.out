Emulation Parameters:
	number to arrive = 10
	r = 1.5
	B = 2
	tsfile = w2data/f1.txt
00000000.000ms: emulation begins
00000679.133ms: token t1 arrives, token bucket now has 1 token 
00001347.241ms: token t2 arrives, token bucket now has 2 tokens
00002554.317ms: p1 arrives, needs 2 tokens, inter-arrival time = 2554.317ms
00002554.325ms: p1 enters Q1
00002554.329ms: p1 leaves Q1, time in Q1 = 0.002ms, token bucket now has 0 token 
00002554.332ms: p1 enters Q2
00002554.354ms: p1 leaves Q2, time in Q2 = 0.022ms
00002554.361ms: p1 begin service at S1, requesting 518ms of service
00002707.030ms: token t5 arrives, token bucket now has 1 token 
00003089.224ms: p1 departs from S1, servie time = 534.863ms, time in system = 534.907ms
00003379.206ms: token t6 arrives, token bucket now has 2 tokens
00006825.472ms: p2 arrives, needs 1 tokens, inter-arrival time = 4271.155ms
00006825.480ms: p2 enters Q1
00006825.484ms: p2 leaves Q1, time in Q1 = 0.002ms, token bucket now has 1 token 
00006825.487ms: p2 enters Q2
00006825.514ms: p2 leaves Q2, time in Q2 = 0.027ms
00006825.518ms: p2 begin service at S1, requesting 387ms of service
00007229.832ms: p2 departs from S1, servie time = 404.314ms, time in system = 404.360ms
00007457.323ms: token t17 arrives, token bucket now has 2 tokens
00008975.768ms: p3 arrives, needs 1 tokens, inter-arrival time = 2150.296ms
00008975.777ms: p3 enters Q1
00008975.781ms: p3 leaves Q1, time in Q1 = 0.002ms, token bucket now has 1 token 
00008975.783ms: p3 enters Q2
00008975.803ms: p3 leaves Q2, time in Q2 = 0.020ms
00008975.807ms: p3 begin service at S1, requesting 3686ms of service
00009080.864ms: p4 arrives, needs 3 tokens, inter-arrival time = 105.096ms, dropped
00009467.838ms: token t22 arrives, token bucket now has 2 tokens
00009565.095ms: p5 arrives, needs 1 tokens, inter-arrival time = 484.231ms
00009565.102ms: p5 enters Q1
00009565.105ms: p5 leaves Q1, time in Q1 = 0.001ms, token bucket now has 1 token 
00009565.108ms: p5 enters Q2
00009565.120ms: p5 leaves Q2, time in Q2 = 0.012ms
00009565.130ms: p5 begin service at S2, requesting 2664ms of service
00010150.790ms: token t23 arrives, token bucket now has 2 tokens
00012245.826ms: p5 departs from S2, servie time = 2680.696ms, time in system = 2680.731ms
00012530.820ms: p6 arrives, needs 2 tokens, inter-arrival time = 2965.725ms
00012530.828ms: p6 enters Q1
00012530.831ms: p6 leaves Q1, time in Q1 = 0.001ms, token bucket now has 0 token 
00012530.834ms: p6 enters Q2
00012530.848ms: p6 leaves Q2, time in Q2 = 0.014ms
00012530.852ms: p6 begin service at S2, requesting 10000ms of service
00012678.362ms: p3 departs from S1, servie time = 3702.555ms, time in system = 3702.594ms
00012853.543ms: token t30 arrives, token bucket now has 1 token 
00013526.006ms: token t31 arrives, token bucket now has 2 tokens
00015176.476ms: p7 arrives, needs 4 tokens, inter-arrival time = 2645.656ms, dropped
00017309.661ms: p8 arrives, needs 4 tokens, inter-arrival time = 2133.185ms, dropped
00019169.225ms: p9 arrives, needs 1 tokens, inter-arrival time = 1859.564ms
00019169.234ms: p9 enters Q1
00019169.237ms: p9 leaves Q1, time in Q1 = 0.001ms, token bucket now has 1 token 
00019169.240ms: p9 enters Q2
00019169.255ms: p9 leaves Q2, time in Q2 = 0.015ms
00019169.259ms: p9 begin service at S1, requesting 1414ms of service
00019662.535ms: token t48 arrives, token bucket now has 2 tokens
00019687.347ms: p10 arrives, needs 1 tokens, inter-arrival time = 518.122ms
00019687.356ms: p10 enters Q1
00019687.358ms: p10 leaves Q1, time in Q1 = 0.001ms, token bucket now has 1 token 
00019687.361ms: p10 enters Q2
00020344.661ms: token t49 arrives, token bucket now has 2 tokens
00020593.624ms: p9 departs from S1, servie time = 1424.365ms, time in system = 1424.399ms
00020593.636ms: p10 leaves Q2, time in Q2 = 906.275ms
00020593.639ms: p10 begin service at S1, requesting 2562ms of service
00022548.214ms: p6 departs from S2, servie time = 10017.362ms, time in system = 10017.394ms
00023174.818ms: p10 departs from S1, servie time = 2581.179ms, time in system = 3487.471ms
00023174.926ms: emulation ends
	average packet inter-arrival time time = 1.96873
	average packet service time = 3.04933

	average number of packets in Q1 = 4.315e-07
	average number of packets in Q2 = 0.0391106
	average number of packets in S1 = 0.373131
	average number of packets in S2 = 0.547922
	average time a packet spent in system = 3.17884
	standard deviation for time spent in system = 3.05075

	token drop probability = 0
	packet drop probability = 0.3

