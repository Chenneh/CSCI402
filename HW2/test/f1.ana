
==== Packets ====
p1: inter-arrival time should be 2544.262ms
    time in Q1 should be 0.003ms
    time in Q2 should be 0.021ms
    should request 518ms of service time
    time in S1 should be 531.019ms
    time in system should be 531.062ms

p2: inter-arrival time should be 4267.873ms
    time in Q1 should be 0.003ms
    time in Q2 should be 0.021ms
    should request 387ms of service time
    time in S1 should be 399.396ms
    time in system should be 399.435ms

p3: inter-arrival time should be 2150.493ms
    time in Q1 should be 0.002ms
    time in Q2 should be 0.020ms
    should request 3686ms of service time
    time in S1 should be 3688.798ms
    time in system should be 3688.836ms

p4: inter-arrival time should be 116.848ms

p5: inter-arrival time should be 508.110ms
    time in Q1 should be 0.002ms
    time in Q2 should be 0.012ms
    should request 2664ms of service time
    time in S2 should be 2669.688ms
    time in system should be 2669.724ms

p6: inter-arrival time should be 2958.342ms
    time in Q1 should be 0.003ms
    time in Q2 should be 0.014ms
    should request 10000ms of service time
    time in S2 should be 10016.589ms
    time in system should be 10016.620ms

p7: inter-arrival time should be 2645.897ms

p8: inter-arrival time should be 2134.531ms

p9: inter-arrival time should be 1871.847ms
    time in Q1 should be 0.002ms
    time in Q2 should be 0.016ms
    should request 1414ms of service time
    time in S1 should be 1430.740ms
    time in system should be 1430.773ms

p10: inter-arrival time should be 521.615ms
    time in Q1 should be 0.002ms
    time in Q2 should be 909.138ms
    should request 2562ms of service time
    time in S1 should be 2580.207ms
    time in system should be 3489.379ms

==== Statistics ====
average packet inter-arrival time = 1.9719818
average packet service time = 3.04520528571429
average number of packets in Q1 = 7.32464813898356e-07
average number of packets in Q2 = 0.039175751312857
average number of packets in S1 = 0.371840502253708
average number of packets in S2 = 0.546603030698117
average time a packet spent in system = 3.17511842857143
standard deviation for time spent in system = 3.05105196372693
token drop probability = 0.791666666666667
packet drop probability = 0.3
