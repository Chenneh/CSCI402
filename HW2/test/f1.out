Emulation Parameters:
	number to arrive = 10
	r = 1.5
	B = 2
	tsfile = w2data/f1.txt
00000000.000ms: emulation begins
00000667.262ms: token t1 arrives, token bucket now has 1 token 
00001346.080ms: token t2 arrives, token bucket now has 2 tokens
00001346.089ms: token t3 arrives, dropped
00002028.429ms: token t4 arrives, dropped
00002544.262ms: p1 arrives, needs 2 tokens, inter-arrival time = 2544.262ms
00002544.270ms: p1 enters Q1
00002544.273ms: p1 leaves Q1, time in Q1 = 0.003ms, token bucket now has 0 token 
00002544.276ms: p1 enters Q2
00002544.297ms: p1 leaves Q2, time in Q2 = 0.021ms
00002544.305ms: p1 begin service at S1, requesting 518ms of service
00002710.491ms: token t5 arrives, token bucket now has 1 token 
00003075.324ms: p1 departs from S1, servie time = 531.019ms, time in system = 531.062ms
00003392.783ms: token t6 arrives, token bucket now has 2 tokens
00003392.791ms: token t7 arrives, dropped
00004072.344ms: token t8 arrives, dropped
00004072.346ms: token t9 arrives, dropped
00004742.332ms: token t10 arrives, dropped
00004742.334ms: token t11 arrives, dropped
00005429.018ms: token t12 arrives, dropped
00005429.021ms: token t13 arrives, dropped
00006110.208ms: token t14 arrives, dropped
00006110.211ms: token t15 arrives, dropped
00006789.638ms: token t16 arrives, dropped
00006812.135ms: p2 arrives, needs 1 tokens, inter-arrival time = 4267.873ms
00006812.144ms: p2 enters Q1
00006812.147ms: p2 leaves Q1, time in Q1 = 0.003ms, token bucket now has 1 token 
00006812.149ms: p2 enters Q2
00006812.170ms: p2 leaves Q2, time in Q2 = 0.021ms
00006812.174ms: p2 begin service at S1, requesting 387ms of service
00007211.570ms: p2 departs from S1, servie time = 399.396ms, time in system = 399.435ms
00007475.133ms: token t17 arrives, token bucket now has 2 tokens
00007475.142ms: token t18 arrives, dropped
00008158.406ms: token t19 arrives, dropped
00008158.409ms: token t20 arrives, dropped
00008847.867ms: token t21 arrives, dropped
00008962.628ms: p3 arrives, needs 1 tokens, inter-arrival time = 2150.493ms
00008962.637ms: p3 enters Q1
00008962.639ms: p3 leaves Q1, time in Q1 = 0.002ms, token bucket now has 1 token 
00008962.642ms: p3 enters Q2
00008962.662ms: p3 leaves Q2, time in Q2 = 0.020ms
00008962.666ms: p3 begin service at S1, requesting 3686ms of service
00009079.476ms: p4 arrives, needs 3 tokens, inter-arrival time = 116.848ms, dropped
00009519.977ms: token t22 arrives, token bucket now has 2 tokens
00009587.586ms: p5 arrives, needs 1 tokens, inter-arrival time = 508.110ms
00009587.594ms: p5 enters Q1
00009587.596ms: p5 leaves Q1, time in Q1 = 0.002ms, token bucket now has 1 token 
00009587.599ms: p5 enters Q2
00009587.611ms: p5 leaves Q2, time in Q2 = 0.012ms
00009587.622ms: p5 begin service at S2, requesting 2664ms of service
00010201.421ms: token t23 arrives, token bucket now has 2 tokens
00010201.429ms: token t24 arrives, dropped
00010868.226ms: token t25 arrives, dropped
00010868.241ms: token t26 arrives, dropped
00011552.383ms: token t27 arrives, dropped
00011552.385ms: token t28 arrives, dropped
00012235.034ms: token t29 arrives, dropped
00012257.310ms: p5 departs from S2, servie time = 2669.688ms, time in system = 2669.724ms
00012545.928ms: p6 arrives, needs 2 tokens, inter-arrival time = 2958.342ms
00012545.936ms: p6 enters Q1
00012545.939ms: p6 leaves Q1, time in Q1 = 0.003ms, token bucket now has 0 token 
00012545.942ms: p6 enters Q2
00012545.956ms: p6 leaves Q2, time in Q2 = 0.014ms
00012545.959ms: p6 begin service at S2, requesting 10000ms of service
00012651.464ms: p3 departs from S1, servie time = 3688.798ms, time in system = 3688.836ms
00012903.687ms: token t30 arrives, token bucket now has 1 token 
00013589.482ms: token t31 arrives, token bucket now has 2 tokens
00013589.492ms: token t32 arrives, dropped
00014281.442ms: token t33 arrives, dropped
00014281.444ms: token t34 arrives, dropped
00014965.148ms: token t35 arrives, dropped
00015191.825ms: p7 arrives, needs 4 tokens, inter-arrival time = 2645.897ms, dropped
00014965.150ms: token t36 arrives, dropped
00015649.153ms: token t37 arrives, dropped
00015649.164ms: token t38 arrives, dropped
00016333.744ms: token t39 arrives, dropped
00016333.747ms: token t40 arrives, dropped
00017002.976ms: token t41 arrives, dropped
00017326.356ms: p8 arrives, needs 4 tokens, inter-arrival time = 2134.531ms, dropped
00017002.979ms: token t42 arrives, dropped
00017686.440ms: token t43 arrives, dropped
00017686.443ms: token t44 arrives, dropped
00018361.743ms: token t45 arrives, dropped
00018361.746ms: token t46 arrives, dropped
00019071.111ms: token t47 arrives, dropped
00019198.203ms: p9 arrives, needs 1 tokens, inter-arrival time = 1871.847ms
00019198.212ms: p9 enters Q1
00019198.214ms: p9 leaves Q1, time in Q1 = 0.002ms, token bucket now has 1 token 
00019198.217ms: p9 enters Q2
00019198.233ms: p9 leaves Q2, time in Q2 = 0.016ms
00019198.236ms: p9 begin service at S1, requesting 1414ms of service
00019719.818ms: p10 arrives, needs 1 tokens, inter-arrival time = 521.615ms
00019719.844ms: p10 enters Q1
00019719.846ms: p10 leaves Q1, time in Q1 = 0.002ms, token bucket now has 0 token 
00019719.849ms: p10 enters Q2
00019739.293ms: token t48 arrives, token bucket now has 1 token 
00020628.976ms: p9 departs from S1, servie time = 1430.740ms, time in system = 1430.773ms
00020628.987ms: p10 leaves Q2, time in Q2 = 909.138ms
00020628.990ms: p10 begin service at S1, requesting 2562ms of service
00022562.548ms: p6 departs from S2, servie time = 10016.589ms, time in system = 10016.620ms
00023209.197ms: p10 departs from S1, servie time = 2580.207ms, time in system = 3489.379ms
00023209.306ms: emulation ends

Statistics:

	average packet inter-arrival time time = 1.9719818
	average packet service time = 3.0452053

	average number of packets in Q1 = 7.3246309e-07
	average number of packets in Q2 = 0.039175751
	average number of packets in S1 = 0.3718405
	average number of packets in S2 = 0.54660303
	average time a packet spent in system = 3.1751184
	standard deviation for time spent in system = 3.051052

	token drop probability = 0.79166667
	packet drop probability = 0.3

